alerts.md

```diff
@@ -165,6 +169,93 @@ Example Helm `values.yaml`:
 
 ----
 
+
+### Type: Asset Budget
+
+Define asset budgets and alert on asset that overruns the threshold set.
+
+*Required parameters:*
+* `type: assetBudget`
+* `threshold: <amount>` -- cost threshold in configured currency units
+* `aggregation: <agg-parameter>` -- configurable, accepts all aggregations supported by the [asset API](https://github.com/kubecost/docs/blob/2ea9021e8530369d53184ea5382b2e4c080bb426/assets.md)
+* `filter: <value>(,<value>,...)` -- configurable, accepts any 1 or more filter values which are comma seperated values
+* `window: <N>d` or `<M>h` -- configurable, (1 ≤ N ≤ 7, 1 ≤ M ≤ 24)
+
+*Valid Aggregation Parameters*: 
+* `category`
+* `cluster`
+* `service`
+* `type`
+* `provider`
+* `account`
+* `providerid`
+* `label` requires the following format: `label:<label_name>=<label_value>`
+
+Example Helm `values.yaml`:
+
+```
+# Daily namespace budget alert on type `Node,Loadbalancer`
+- type: assetBudget
+  threshold: 50
+  window: daily # or 1d
+  aggregation: type
+  filter: Node,LoadBalancer
+# 3d cluster budget alert on cluster with no filter set
+- type: assetBudget
+  threshold: 100
+  window: 3d
+  aggregation: cluster
+  filter: ''
+```
+
+----
+
+### Type: Cloud Report
+
+Sends a recurring email and/or Slack alert with a asset summary report of cost and efficiency metrics. 
```

Comment:
"an cloud or Kubernetes asset summary report"?

Reply:
Let me know if "Sends a recurring email and/or Slack alert with a cloud and/or Kubernetes assets summary report. " is good.